lib/book.rb

```diff
@@ -1,2 +1,14 @@
+class Book 
+  attr_reader :title
+  attr_accessor :author
+  attr_accessor :page_count
+  attr_accessor :genre
```

Comment:
You can also do this:
```suggestion
  attr_accessor :author, :page_count, :genre
```